docs/src/main/asciidoc/kubernetes-client.adoc

```diff
@@ -73,7 +73,30 @@ To make testing against a mock Kubernetes API extremely simple, Quarkus provides
 a mock of the Kubernetes API server and sets the proper environment variables needed so that the Kubernetes Client configures itself to use said mock.
 Tests can inject the mock and set it up in any way necessary for the particular testing using the `@MockServer` annotation.
 
-Here is an example of such test:
+Lets assume we have a REST endpoint defined like so:
+
+[source%nowrap,java]
+----
+
+@Path("/pod")
+public class Pods {
+
+    private final KubernetesClient kubernetesClient;
+
+    public Pods(KubernetesClient kubernetesClient) {
+        this.kubernetesClient = kubernetesClient;
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/{namespace}")
+    public List<Pod> pods(@PathParam("namespace") String namespace) {
+        return kubernetesClient.pods().inNamespace(namespace).list().getItems();
+    }
+}
+----
+
+We could write a test for this endpoint very easily like so:
```

Comment:
Does the URL used in the test matches the URL in the resource?

It doesn't look like it but maybe I'm missing something?

Reply:
The RestAssured test and the JAX-RS endpoint do seem to match. The other URLs are the Kubernetes API URLs that are mocked